wled00/wled.h

```diff
@@ -769,6 +769,9 @@ WLED_GLOBAL volatile uint8_t jsonBufferLock _INIT(0);
 #define W(c) (byte((c) >> 24))
 
 class WLED {
+private:
```

Comment:
I'm working on 3 usermods that require this sort of thing, one of which is this exact feature set, bluetooth json api, by means of a different method. This PR beat me to the submission.
So I will say that locking this behind a single usermod define would be troublesome for other usermods.